index.html

```diff
@@ -0,0 +1,141 @@
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.typekit.net/uzf0rar.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <title>Reneweable Energy Conference</title>
+</head>
+<body>
+    <section class="background">
+        <nav class ="desktopnav">
+            <div class="space"></div>
+            <ul class="socialmedia">
+                <li><a href="#"><i class=" fa fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class=" fa fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <nav class ="desktop2nav">
+            <div class="space2"><img src="image/logo-EHS.png" class="desktoplogo" alt="clean Energy Logo"></div>
+            <ul class="socialmedia2">
+                <li><a href="about.html" id="about">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#" id="Campaign">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="meuncomponents">
+            <button class="menubtn" name ="menu button"type="submit"><img src="image/hamburger.JPG" id ="menu" alt="menu button"></button>
+            <div class="menuspace"></div>
+        </div>
+        <div class="popup">
+            <a href="about.html"> About </a>
+        </div>
+        <div class="header-text">
+            <h2>"Hello! Sharing World"</h2>
+                <h1>National Reneweable Energy Submit 2023</h1>
+           </div>
+           <div class="header-paragraph">
+            <p>A global submit where experts in the field of Reneweable Energy Technologies from all over the world would converge to discuss the latest developments in Lagos State Nigeria, March 2023</p>
+           </div>
+
+           <div class="header-footer">
+            <h2>2023.03.01(WED)~02(THU)</h2>
+            <p> @National art theater iganmu lagos</p>
+           </div>
+        </section>
+    <section class="main-program">
+            <article class="main-heading">
+                <h2>Main Program</h2>
+                <hr>
+            </article>
+            <article class= "main-itemcontainer">
+            <article class="main-item">
+                <article class="mainIMGH">
+                    <img class="main-image" src="image/speaker.png" alt="Lecture">
+                    <h2>Lecture</h2>
+                </article>
+                
+                <p>Listen to various experts in Reneweable Energy share that ideas on the future of Reneweable</p>
+            </article>
+            <article class="main-item">
+                <article class="mainIMGH">
```

Comment:
- [ ] **[OPTIONAL]** You did a nice job applying the use of `<article>` tags instead of divs. However, this doesn't always apply. The `<article>` tag is widely used for elements that are self-contained compositions, in this case, is a great option for each main-program card! 🤓 

But, following the definition, using `<article>` for the image and header isn't quite right, because it depends on the existence of a card to exist. So please consider fixing this! 🤓 🔧 


My suggested markup is this one:

```html
<div class="main-itemcontainer">
  <article class="main-item">
    <div>
      <img />
      <h2>...</h2>
    </div>
    <p> ... </p>
  </article>
...
</div>

```

Read more about this [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).